api/v1/submissions/PKPSubmissionController.php

```diff
@@ -1050,8 +1050,7 @@ public function getPublications(Request $illuminateRequest): JsonResponse
 
         $currentUserReviewAssignment = Repo::reviewAssignment()->getCollector()
             ->filterBySubmissionIds([$submission->getId()])
-            ->filterByReviewerIds([$request->getUser()->getId()])
-            ->filterByLastReviewRound(true)
+            ->filterByReviewerIds([$request->getUser()->getId()], true)
```

Comment:
Fix the issue with broken anonymity. Check the latest review round per reviewer per submission instead of just per submission